src/sql/plot.py

```diff
@@ -1,35 +1,29 @@
 """
 Plot using the SQL backend
 """
-import functools
-
-import matplotlib.pyplot as plt
-import numpy as np
+from ploomber_core.dependencies import requires
 from jinja2 import Template
 
-from sql.store import store
-import sql.connection
-
-# TODO: support for a select statement to define table and column
+try:
+    import matplotlib.pyplot as plt
+except ModuleNotFoundError:
+    plt = None
 
-# %%time
-# %%memit
+try:
+    import numpy as np
+except ModuleNotFoundError:
+    np = None
 
 
-def log(func):
-    @functools.wraps(func)
-    def wrapper(*args, **kwargs):
-        # print(f'start: {func.__name__}')
-        result = func(*args, **kwargs)
-        # print(f'end: {func.__name__}')
-        return result
+from sql.store import store
+import sql.connection
 
-    return wrapper
+# TODO: support for a select statement to define table and column
 
 
 # TODO: test with NAs
```

Comment:
Add the missing test?

Reply:
I was testing indirectly when building the docs, but you're right, better to have explicit tests. I added them to `test_magic_plot.py`